daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/dsom/SequenceGroup.scala

```diff
@@ -296,6 +259,65 @@ class Sequence(xmlArg: Node, parent: SchemaComponent, position: Int)
 
 }
 
+trait SequenceLikeMixin
```

Comment:
scaladoc. Also consider renaming this mixin to be more specific. I think this is SequenceDefMixin, and it captures that the component carries a DFDLSequence annotation (not a DFDLGroup annotation), and contains XML elements inside it.